content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -78,19 +78,36 @@ and allows you to manage the stock of your products under the FBA program.
 Issue invoices and register payments
 ====================================
 
-You can issue invoices for Amazon orders in Odoo. Click **Create Invoice** in the sales order to do
-so. You can also do it in batch from the list view of orders. Then, confirm and send the invoices to
-your customers.
+Issue invoices
+--------------
+
+Sending invoices to Amazon customers directly from Odoo is not feasible due to Amazon's policy of
+not sharing customer email addresses. Instead, you can manually upload the
+invoices generated on Odoo to the Amazon backend.
+
+In addition, it is currently required to manually retrieve VAT numbers from the Amazon backend
+before creating an invoice due to an issue with the Selling Partner API.
+
+.. note::
+   For :doc:`TaxCloud <../../../finance/accounting/taxes/taxcloud>` users: invoices created from
+   Amazon sales orders are **not** synchronized with TaxCloud since Amazon already includes them in
+   its own tax report to TaxCloud.
+
+Register payments
+-----------------
+
+As customers pay Amazon as an intermediary, creating a dedicated *Bank* journal with a dedicated
+*Bank and Cash* intermediary account is recommended.
```

Comment:
Im not sure to understand this and what we are trying to recommend and it does not sound logical with "Amazon journal" used after in the text

Reply:
I will clarify by suggesting a name for this journal.
Christophe said the flow was ok (content taken from the current documentation).